src/@next/Switch/Switch.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { Meta, Story } from '@storybook/react';
+import React from 'react';
+
+import { BaseContainer } from '../../Layout/GlintsContainer/GlintsContainer';
+import { Switch } from './Switch';
+
+(Switch as React.FunctionComponent).displayName = 'Switch';
+
+const defaultText = 'Badge';
```

Comment:
<img width="415" alt="image" src="https://user-images.githubusercontent.com/52077891/219615956-7d6acac6-fb7b-4f01-b6dd-69c0de90cccc.png">

Check all of these comment from bot

Reply:
cleared all warnings